test/e2e/config/docker.yaml

```diff
@@ -234,7 +234,6 @@ providers:
     - sourcePath: "../data/infrastructure-docker/main/cluster-template-machine-pool.yaml"
     - sourcePath: "../data/infrastructure-docker/main/cluster-template-node-drain.yaml"
     - sourcePath: "../data/infrastructure-docker/main/cluster-template-upgrades.yaml"
-    - sourcePath: "../data/infrastructure-docker/main/cluster-template-upgrades-cgroupfs.yaml"
```

Comment:
I'm probably missing something but this template is still used in the cluster upgrade test for Kubernetes < 1.24

This basically depends on Kubernetes version not on kind. I think we can get rid of this once MachinePools are supported in ClusterClass (we only need this to set the cgroupdriver correctly for MachinePools, for MDs it's handled in the ClusterClass)

Reply:
I have grokked for this template but have not found it, could you point me to where it is used?

Reply:
Don't bother, found it, I'm going to revert